### PR TITLE
chore: update module

### PR DESCRIPTION
**Pull request by ricardogobbosouza**
- Remove babel config
- Update badges
- Update dependencies
- Update tests
- Resolves #34, resolves #33, resolves #32, resolves #30

### CODE REVIEW

**Comment by ricardogobbosouza** [comment]
ping @manniL

File: .eslintrc.js [hunk 1]
```diff
@@ -1,9 +1,4 @@
 module.exports = {
-  root: true,
```
Comment (TheAlexLichter) [inline .eslintrc.js]: Why did you remove that part of the ESLint config? :thinking:

Comment (TheAlexLichter) [reply .eslintrc.js]: Other than that, LGTM.

Comment (ricardogobbosouza) [reply .eslintrc.js]: You're right, I have to set this on the other modules too! damn it

Comment (TheAlexLichter) [reply .eslintrc.js]: What's with

`parserOptions: {
    parser: 'babel-eslint',
    sourceType: 'module'
  },` :thinking: 	

Using babel-eslint is crucial in case the module is using ES proposals that are not part of the language yet.

Comment (ricardogobbosouza) [reply .eslintrc.js]: In this module there is nothing that needs `babel-eslint`

Comment (TheAlexLichter) [reply .eslintrc.js]: Yes, not in this module :relaxed: 	I meant with respect to other modules

Comment (ricardogobbosouza) [reply .eslintrc.js]: Ah yes, in some it is necessary
